services/auth/authServices.ts

```diff
@@ -0,0 +1,27 @@
+import { AxiosResponse } from "axios";
+import api from "../../shared/fetcher";
+import { User } from "../initChatAnonymous/initChatAnonymousServices";
```

Comment:
please change relative import to alias import
e.g
import { BannerPage } from "component/Page/BannerPage"